src/RuleSet/Sets/PhpCsFixerRiskySet.php

```diff
@@ -47,6 +47,7 @@ public function getRules(): array
             ],
             'no_unreachable_default_argument_value' => true,
             'no_unset_on_property' => true,
+            'php_unit_data_provider_static' => true,
```

Comment:
I believe PhpCsFixer set doesn't need to include PHPUnitMigration set. let use use the both sets in config